test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs

```diff
@@ -346,6 +346,19 @@ public void Migration_outputs_error_when_no_projects_found(bool useGlobalJson)
             result.StdErr.Should().Contain("Migration failed.");
         }
 
+        [Fact]
+        public void It_migrates_runtimes_into_csproj()
```

Comment:
It does more than that. It migrates and publishes projects with runtimes. We need to update this test name.